hudi-common/src/main/java/org/apache/hudi/common/model/HoodieAvroIndexedRecord.java

```diff
@@ -162,7 +162,7 @@ public HoodieRecord wrapIntoHoodieRecordPayloadWithParams(
       Option<String> partitionNameOp,
       Boolean populateMetaFields,
       Option<Schema> schemaWithoutMetaFields) {
-    String payloadClass = ConfigUtils.getPayloadClass(props);
+    String payloadClass = ConfigUtils.getAvroPayloadClass(props);
```

Comment:
I think we can keep the `getPayloadClass` naming.  The payload class implicitly indicates it's using the Avro record.

Reply:
changed back